package.json

```diff
@@ -203,10 +203,10 @@
     "ci": "yarn ci-frontend && yarn ci-backend",
     "ci-frontend": "yarn lint && yarn test",
     "ci-backend": "lein docstring-checker && lein bikeshed && lein eastwood && lein test",
-    "test-cypress": "yarn build && ./bin/build-for-test && yarn test-cypress-no-build",
+    "test-cypress": "yarn build && ./bin/build-for-test && yarn test-cypress-no-build && yarn test-cypress-smoketest",
     "test-cypress-open": "./bin/build-for-test && yarn test-cypress-no-build --open",
     "test-cypress-no-build": "yarn && CONFIG_FILE=frontend/test/cypress.json babel-node ./frontend/test/__runner__/run_cypress_tests.js",
-    "test-cypress-smoketest": "yarn run test-cypress-no-build --testFiles frontend/test/metabase-smoketest"
+    "test-cypress-smoketest": "yarn test-cypress-no-build --testFiles frontend/test/metabase-smoketest"
```

Comment:
@ariya is there a way to refine `test-cypress`? Because of the way we include `--testFiles`, I didn't know of any other way than to add smoke tests **on top** of the previous commands. It leads to some repetition:

`yarn build && ./bin/build-for-test && yarn test-cypress-no-build && yarn test-cypress-no-build --testFiles frontend/test/metabase-smoketest`

Reply:
I think your proposed change in this PR is clear enough, it's easy to follow even with the said repetition.